test/visual/report.cpp

```diff
@@ -79,12 +84,34 @@ unsigned console_report::summary(result_list const & results)
             case STATE_ERROR: error++; break;
             case STATE_OVERWRITE: overwrite++; break;
         }
+
+        duration_map_type::iterator duration = durations.find(r.renderer_name);
+        if (duration == durations.end())
+        {
+            durations.emplace(r.renderer_name, r.duration);
+        }
+        else
+        {
+            duration->second += r.duration;
+        }
```

Comment:
Does this have some overhead? I wonder if it should also be conditional on `if (show_duration)`?